app/config/region.ts

```diff
@@ -0,0 +1,6 @@
+const region = {
+	locale: "en-US",
+	timeZone: "UTC",
+} as const;
+
+export default region;
```

Comment:
[Style] How certain are you that you prefer `export default` in `.ts`/`.tsx` files? 😄 I'm pretty -1 on it in general, for the reasons in the first ~half of https://basarat.gitbook.io/typescript/main-1/defaultisbad.

What are your thoughts on switching to `export const site = ...`? _(and maybe combining into `app/config.ts`, since the files are pretty small as-is?)

Alternately, I also don't see a strong delineation between `app/config/*.ts` and `app/data/*.json`. Maybe these could all be `.json` files, ironically which would then have to be `default`-imported? 

FWIW my personal instinct is to go with a style that results in imports like `import { region } from "~/config"`. But I'm definitely not the only voice here that should be heard.

Reply:
The link you provided had strong opinions I can understand and respect.
My take is that default exports/imports are a good way to define the primary purpose of a module, so it's my go-to for files with a single goal. 
It helps me be focused when structuring different app models, directory structures, etc...
This is why I also chose a design of single-focused files.

On the one hand, when default exports are handled correctly, they can maintain very focused modules toward the app.
On the other hand, as the article you provided explains, it can go wrong when they are not handled in a specific manner.

I have two suggestions in mind:
1. Take your approach for a single config file - it can keep our imports aligned; The main drawback is that a single config file can grow over-limit, though it may not be very likely for this app.
2. Add a "barrel" `index.ts` file for exporting each module in the config directory so that each module will be exported as named export.

I think it might be more likely to take the first approach honestly, but we can have a discussion around it

Reply:
The last thing I missed regarding moving to a `.json` config file over `.ts`: 
The main benefit is that in .ts I can use `as const` to get a better IntelliSense for consts, which is neat :)

<img width="444" alt="image" src="https://user-images.githubusercontent.com/45559220/235803440-538e2450-7141-43b9-b63f-d3d32e95b3ca.png">

Reply:
> single config file

Yeah that'd be my preference. Barrel (`index.ts`) re-exports are nice when you have a lot, but we only have a couple files. Seems premature to me.

> `as const`

+1 that is a great benefit! 😄